traits/trait_base.py

```diff
@@ -33,6 +34,8 @@
 
 SequenceTypes = (list, tuple)
 
+EnumTypes = (list, tuple, set, enum.EnumMeta)
```

Comment:
`enum.EnumMeta` doesn't seen right here: an instance of `enum.EnumMeta` is an enumeration _class_, where presumably we want to check for an instance of an enumeration class. Do you just want `enum.Enum` here?

Reply:
> Do you just want `enum.Enum` here?

No, you don't. I misunderstood how this was being used. Closing.